### DISCUSSION TITLE
How do I change theme?

### DISCUSSION THREAD

**Discussion by Chris3y** [reduxjs/redux-devtools#1234] [General]
This trace feature is amazing but the current theme makes stuff a little tough to read. Can I change it?

![image](https://user-images.githubusercontent.com/13569218/193548291-8caab9f7-b478-47a3-af92-36b3dde66e79.png)

**Comment by Chris3y** [comment]
For anyone looking, I found a workaround in that, if you go to the Settings tab then Themes and set the tools to a dark theme, things read much more nicely and the very dark text becomes very light.

**Reply by nikonov91-dev** [reply]
i changed to dark, but still not visible
